generation/3d_ldm/3d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
<p>To be consistent with other notebook, let's use <code>## Cleanup data directory</code> here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>